src/acp.js

```diff
@@ -91,14 +91,26 @@ function handleGlobalBanDelete(user, data) {
 }
 
 function handleListUsers(user, data) {
-    var name = data.name;
-    if (typeof name !== "string") {
-        name = "";
+    var query = data.query;
```

Comment:
If I'm interpreting your code correctly, the "field" is the column being searched, whereas what you're calling "query" here is the value to look for.  Is that right?